### PR TITLE
Update Dutch translations

### PR DESCRIPTION
**Pull request by robdekort**

### CODE REVIEW

File: resources/lang/nl/permissions.php [hunk 1]
```diff
@@ -71,4 +71,6 @@
     'access_utility' => ':title',
     'access_utility_desc' => 'Geeft toegang tot het :title gereedschap',
     'group_misc' => 'Diversen',
+    'resolve_duplicate_ids' => 'Los dubbele ID''s op',
+    'resolve_duplicate_ids_desc' => 'Geeft de mogelijkheid om dubbele ID''s te zien en op te lossen.',
```
Comment (jasonvarga) [inline resources/lang/nl/permissions.php:75]: The apostrophes in here cause PHP syntax errors. Did you add these manually or did the CLI tool cause them?

**Comment by robdekort** [comment]
Oh darn. Manually! Want me to fix this?

Verstuurd vanaf mijn iPhone

**Comment by jasonvarga** [comment]
Nope, fixed already. Thanks. 👍

**Comment by robdekort** [comment]
You’re awesome. Got confused with the yaml syntax. Funny thing is my editor showed this as valid…

Verstuurd vanaf mijn iPhone

**Pull request merged**